Apps/Src/ReadTritium.c

```diff
@@ -127,3 +121,52 @@ float Motor_RPM_Get(){ //getter function for motor RPM
 float Motor_Velocity_Get(){ //getter function for motor velocity
 	return Motor_Velocity;
 }
+
+
+/**
+ * Error handler functions
+ * Passed as callback functions to the main assertTaskError function by assertTritiumError
+*/
+
+/**
+ * @brief A callback function to be run by the main assertTaskError function for hall sensor errors
+ * restart the motor if the number of hall errors is still less than the RESTART_THRESHOLD.
+ */ 
+static void handler_Tritium_HallError(void) {
+	MotorController_Restart(); 
+}
```

Comment:
```suggestion
static inline void handler_Tritium_HallError(void) {
	MotorController_Restart(); 
}
```

inline will make the compiler just stick the motorcontroller_restart wherever the handler is called, which is a little optimization